README.md

```diff
@@ -27,6 +32,12 @@ Under development
 
 If you discover a security vulnerability within this application, please send an e-mail to security@ark.io. All security vulnerabilities will be promptly addressed.
 
+## Credits
+
+- [FX Thoorens](https://github.com/fix)
+- [Juan A. Martín](https://github.com/j-a-m-l)
```

Comment:
Replace that with `[Joshua Noack](https://github.com/supaiku0)` and it's ready to be merged.